judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/test/TestDefinition.java

```diff
@@ -0,0 +1,32 @@
+package hu.blackbelt.judo.test;
+
+/*-
+ * #%L
+ * JUDO Runtime Core JSL :: Parent
+ * %%
+ * Copyright (C) 2018 - 2022 BlackBelt Technology
+ * %%
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention( RetentionPolicy.SOURCE )
+@Target( value = { ElementType.METHOD, ElementType.TYPE })
+public @interface TestDefinition {
```

Comment:
I did not find any usage of this annotation in this PR's scope

Reply:
Yes, this is not used yet, but I have deleted.